app/views/about/index.html.erb

```diff
@@ -1,60 +1,74 @@
+    <a class="btn button-primary about-button-primary" href="/tos">Terms of Service</a>
+    <a class="btn button-primary about-button-primary" href="/privacy">Privacy Policy</a>
+  </div>
+  <div class="row row-text">
+    <div class="col-xs col-sm col-md col-lg">
+      <div class="card-small-social about-card-social-1">
+        <a class="about-social-anchor" href="mailto:support@circuitverse.org" target="_blank">
+          <%= image_tag "SVGs/email.svg", height: "60", width: "60", alt: "Email Icon", class: "card-small-social-image" %>
+          <h6>Email us at</h6>
+          <p>support@circuitverse.org</p>
+        </a>
+      </div>
+      <div class="card-small-social about-card-social-2">
+        <a class="about-social-anchor" href="<%= Rails.configuration.slack_url %>" target="_blank">
+          <%= image_tag "SVGs/slack.svg", height: "60", width: "60", alt: "Slack Logo", class: "card-small-social-image" %>
+          <h6>Join and chat with us at</h6>
+          <p>Slack Channel</p>
+        </a>
+      </div>
+    </div>
+  </div>
+  <hr>
+  <div class="row row-text">
+    <div class="col-xs col-sm col-md col-lg">
+      <h2 class="heading-submain">CONTRIBUTORS</h2>
+      <h5>Meet the awesome people of CircuitVerse community that've made this platform what it is now.</h5>
+    </div>
+  </div>
+  <div class="row row-text">
+    <div class="about-contributors-section">
 
-            </div>
-          </div>
-          <script type="text/javascript">
-            async function getContributors(url){
-              let apiData = await fetch(url);
-              let contributorsData = await apiData.json();
-              for (let i=0; i<contributorsData.length; i++){
-                let contributorElement = document.createElement('div');
-                let contributorElementAnchor = document.createElement('a');
-                let contributorElementImage = document.createElement('img');
-                contributorElement.classList.add('contributor-about');
-                contributorElementAnchor.href = await contributorsData[i]['html_url'];
-                contributorElementImage.src = await contributorsData[i]['avatar_url'];
-                contributorElementImage.classList.add('contributor-image-about');
-                contributorElementAnchor.appendChild(contributorElementImage);
-                contributorElement.appendChild(contributorElementAnchor);
-                document.getElementsByClassName('flex-container-about')[0].appendChild(contributorElement);
-              }
-            }
-            getContributors('https://api.github.com/repos/CircuitVerse/CircuitVerse/contributors');
-          </script>
-        </div>
+    </div>
+  </div>
+  <br>
+  <div class="row row-text">
+    <div class="col-xs col-sm col-md col-lg text-center">
+      <a class="btn button-primary about-button-contribute" href="https://opencollective.com/CircuitVerse" target="_blank">Become a Contributor</a>
```

Comment:
This button would redirect user to the Contribute Page.

Reply:
Done